src/components/SEO/Default.tsx

```diff
@@ -0,0 +1,38 @@
+import { DefaultSeo } from 'next-seo'
+import React from 'react'
+import { NextRouter } from 'next/dist/shared/lib/router/router'
+
+interface SEOHeaderProps {
+  router: NextRouter
+}
+
+const SEOHeader = ({ router }: SEOHeaderProps) => (
+  <DefaultSeo
+    title="Braindao | Largest Blockchain & Crypto Encyclopedia"
```

Comment:
title: BrainDAO |  native DAO and treasury of the IQ Token
description: BrainDAO is the native DAO and treasury of the IQ Token which powers IQ.wiki 

( change in opengraph and other descriptions, also make sure titleTemplate and others uses BrainDAO (not Braindao ))

Reply:
Okay then. Thanks